x-pack/test/security_solution_cypress/config.ts

```diff
@@ -52,6 +52,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'alertDetailsPageEnabled',
         ])}`,
+        '--xpack.cloud.id=test',
```

Comment:
Could you please add a comment that this config line is needed to enable the guided onboarding tour in e2e tests?